src/pages/Contests/index.js

```diff
@@ -0,0 +1,155 @@
+import {
+  AppBar,
+  IconButton,
+  Toolbar,
+  Box,
+  Button,
+  Container,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
+import React, { useState } from "react";
+import ArrowBackIcon from "@material-ui/icons/ArrowBack";
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+import ContestsList from "./ContestsList";
+import contests from "../../data/contestsDetails.json";
+const useStyle = makeStyles((theme) => ({
+  root: {
+    margin: "5px",
+  },
+  buttoncontainer: {
+    marginTop: theme.spacing(1),
+  },
+  backButton: {
+    marginRight: theme.spacing(2),
+  },
+  title: {
+    flexGrow: 1,
+  },
+  appBar: {
+    background: "#160050",
+  },
+  trophyIcon: {
+    height: 30,
+    width: 30,
+  },
+  titlecontainer: {
+    textAlign: "center",
+    marginTop: "5px",
+  },
+  button: {
+    textTransform: "none",
+    borderRadius: 5,
+    color: "#fff",
+    marginRight: theme.spacing(1),
+    marginTop: theme.spacing(3),
+    paddingRight: theme.spacing(2),
+    paddingLeft: theme.spacing(2),
+  },
+  contestContainer: {
+    marginTop: theme.spacing(3),
+  },
+}));
+const ongoing = contests[0].ongoing;
+const upcoming = contests[0].upcoming;
+const completed = contests[0].completed;
+
+function Contests() {
+  const classes = useStyle();
+  const buttonItems = [
+    {
+      status: "ongoing",
+    },
+    {
+      status: "upcoming",
+    },
+    {
+      status: "completed",
+    },
+  ];
+  const [status, setStatus] = useState("Active");
+  const setStatusFilter = (status) => {
+    setStatus(status);
+  };
+  return (
+    <Container component="main" maxWidth="xs" className={classes.root}>
```

Comment:
I think wrapping this container is creating problems with unwanted padding with in appBar

Reply:
okay, I'll fix it.